docs/WritingTests.asciidoc

```diff
@@ -184,14 +180,15 @@ on GitHub] for examples.
 
 Variables are accessible via the *get_var* and *check_var* functions.
 
-Modifying setting of an existing test
--------------------------------------
+== Test Development tricks
+
+=== Modifying setting of an existing test
 
 There is no interface to modify existing tests but the clone script
 can be used to create a new job that adds, removes or changes
 settings:
 
-[source,sh]
+[source,bash]
```

Comment:
ok, it's just a syntax highlighting marker but I always prefer "sh" if one does not actually need bash. `sh` could be more lightweight, i.e. your system starts up faster if you don't need bash ;-)